server/prisma/schema.prisma

```diff
@@ -200,13 +200,30 @@ model Group {
   memberActivity MemberActivity[]
   patronages     Patron[]
   socialLinks    GroupSocialLinks[]
+  roleOrder      GroupRoleOrder[]
 
   // Constraints
   @@unique([name], map: "groups_name_key")
   // Map
   @@map("groups")
 }
 
+model GroupRoleOrder {
+  groupId Int
+  role    GroupRole @unique @default(member)
```

Comment:
no need for default here, it needs to be a required field

Reply:
done